lua/terminal.lua

```diff
@@ -287,8 +287,7 @@ end
 -- @tparam integer buf A value of 0 implies the current buffer.
 -- @tparam[opt=initialize_terminal_colors()] {[number]=string} rgb_color_table cterm color RGB lookup table
 -- @see highlight_buffer
-local function attach_to_buffer(buf, rgb_color_table)
-	rgb_color_table = rgb_color_table or initialize_terminal_colors()
+local function attach_to_buffer(buf)
```

Comment:
I think the real issue is that I forgot to forward `rgb_color_table` to `highlight_buffer` here, not that `rgb_color_table` is unused.